data/liquidvapor.yaml

```diff
@@ -57,14 +57,22 @@ phases:
     T: 300.0
     P: 1.01325e+05
   pure-fluid-name: oxygen
+- name: carbon-dioxide
+  thermo: pure-fluid
+  elements: [C, O]
+  species: [CO2]
+  state:
+    T: 280.0
+    P: 1.01325e+05
+  pure-fluid-name: carbon-dioxide
```

Comment:
Is this intentionally a duplicate of the next entry (except the `name`) or is it a goof in the diff view?

Reply:
Deliberate. I want `carbon-dioxide` to be a valid name, but I don't want to to deprecate the old name yet because it would screw with the tests (specifically, `cti2yamlTest.test_liquidvapor` and `ctml2yamlTest.test_liquidvapor`).

Reply:
But now we have no tests of the old name, is that what you wanted? I guess I'm confused why we wouldn't test something if it's not deprecated yet?

Reply:
Actually, looking at this again, I realized that deprecating these phase names doesn't interfere with those tests after all, so we can do so now. I've just pushed a revised commit for this.

Reply:
Oh brother, I thought this was part of the test suite, not that it was the main `liquidvapor.yaml` input file 🤦‍♂ Well, I'm glad it worked out!